src/Platform/Microsoft.Testing.Platform/PublicAPI/PublicAPI.Unshipped.txt

```diff
@@ -1 +1,2 @@
 #nullable enable
+*REMOVED*[TPEXP]Microsoft.Testing.Platform.Services.IPlatformInformation.BuildDate.get -> System.DateTimeOffset?
```

Comment:
Note: While this is public API, it's experimental and I think we should be fine breaking it. I checked and currently it's used only by MSTest. It's not used by TUnit or xUnit.